mockgcp/mockbigqueryconnection/connection.go

```diff
@@ -202,35 +203,10 @@ func (s *ConnectionV1) UpdateConnection(ctx context.Context, req *pb.UpdateConne
 		return nil, err
 	}
 
-	paths := req.GetUpdateMask().GetPaths()
-	for _, path := range paths {
-		switch path {
-		case "friendlyName":
-			obj.FriendlyName = req.GetConnection().GetFriendlyName()
-		case "description":
-			obj.Description = req.GetConnection().GetDescription()
-		default:
-			return nil, status.Errorf(codes.InvalidArgument, "update_mask path %q not valid", path)
-		}
+	if err := fields.UpdateByFieldMask(obj, req.Connection, req.UpdateMask.Paths); err != nil {
+		return nil, fmt.Errorf("update field_mask.paths: %w", err)
```

Comment:
Nit: arguably if we're using one of our functions we can just `return nil, err` here.  If we're not adding information that is specific to how we are calling UpdateByFieldMask, we might as well put it into the UpdateByFieldMask errors!